src/api/main.py

```diff
@@ -0,0 +1,65 @@
+import os
+from os.path import join
+import errno
+import shutil
+from subprocess import call
+import json
+import uuid
+
+from flask import Flask, jsonify
+
+from rwd.Rapid_Watershed_Delineation import Point_Watershed_Function
+
+app = Flask(__name__)
+
+
+@app.route('/rwd/<lat>/<lon>', methods=['GET'])
+def run_rwd(lat, lon):
+    """
+    Runs RWD for lat/lon and returns the GeoJSON
+    for the computed watershed boundary.
+    """
+    # TODO This function needs to be made more robust to failures.
+
+    data_path = '/opt/rwd-data'
+
+    # Create a temporary directory to hold the output
+    # of RWD
+    output_path = join('/tmp/rwd/', str(uuid.uuid4()))
```

Comment:
Consider using the built in [mkdtemp](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp)